ny-portal/src/app/(frontend)/komponenter/komponenter.module.scss

```
@@ -16,31 +16,116 @@
     width: 100%;
     display: flex;
     flex-direction: column;
+}
 
-    .componentGallery {
-        list-style: none;
-        padding: 0;
-        margin: 0;
-        display: grid;
-        grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
-        gap: jkl.$spacing-24;
+.componentGallery {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+    gap: jkl.$spacing-24;
+    container-type: inline-size;
+
+    &.listView {
+        grid-template-columns: auto;
+        border-top: 1px solid var(--jkl-color-border-separator);
+        gap: 0;
+
+        > * {
+            border-bottom: 1px solid var(--jkl-color-border-separator);
+        }
+    }
+}
+
+.componentCard {
+    display: grid;
+    grid-template-columns: 1fr;
+    gap: var(--jkl-unit-30);
+    width: 100%;
+    text-decoration: none;
+    color: var(--color-text);
 
-        .component {
-            width: 100%;
-            text-decoration: none;
-            color: var(--color-text);
+    .componentGallery.listView & {
+        grid-template-columns: repeat(12, 1fr);
+        gap: var(--jkl-unit-05) var(--jkl-unit-30);
+        align-items: center;
+        padding-block: var(--jkl-unit-30);
+
+        > * {
+            grid-column: 1/-1;
+        }
+
+        .image {
+            grid-column: 1/3;
+        }
+
+        .name {
+            grid-column: 3/6;
+        }
+
+        .description {
+            grid-column: 6/12;
+        }
+
+        &::after {
+            content: "arrow_forward" / "";
+            font-family: "Fremtind Material Symbols";
+            grid-column: 12/13;
+            aspect-ratio: 1;
+            border: 1px solid var(--jkl-color-border-action);
+            border-radius: 9999px;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            width: var(--jkl-unit-30);
+            padding: var(--jkl-unit-10);
+        }
+
+        @container (max-width: 45em) {
+            .image {
+                display: none;
+            }
+            .name,
+            .description {
+                grid-column-start: 1;
+            }
 
             .name {
-                @include jkl.text-style("heading-4");
+                grid-row: 1 / 2;
             }
 
-            .image {
-                width: 100%;
-                height: auto;
-                object-fit: cover;
-                border-radius: var(--jkl-spacing-16);
-                aspect-ratio: 1.6;
+            .description {
+                grid-row: 2 / 3;
```

Flyttet opp fra linje 109

```suggestion
                grid-row: 2 / 3;
                display: block;
                @include jkl.text-style("small");
```